tests/report/reportportal/test.sh

```diff
@@ -376,7 +410,12 @@ rlJournalStart
             test_case_id=$(yq -r ".\"$test_name\".id" test.fmf)
             [[ $test_case_id != null ]] && rlAssertEquals "Assert the test ${test_name} has a correct testCaseId" "$(echo $response | jq -r '.testCaseId')" "$test_case_id"
             echo "$response" | jq -r ".$jq_element" > tmp_attributes.json || rlFail "$jq_element listing into tmp_attributes.json"
-            rlAssertGrep "TMT_TREE" tmp_attributes.json
+
+            # FIXME: Ignore the check for subtests
```

Comment:
a fixme item

Reply:
@seberm @thrix is there anything to address?

Reply:
There are a couple more. Let's fix those during the proper fix of the test in the separate pull request as agreed.